recipe_scrapers/plugins/_interface.py

```diff
@@ -0,0 +1,29 @@
+class PluginInterface:
```

Comment:
Very neat - so the idea is that we can have an ecosystem of (shared, reusable) plugins to handle various types of content, and post-processing of content?

Reply:
Yeah exactly! And most importantly people can write/prototype and attach plugins without the need to touch the core package. So I hope there is some movement in that aspect.